tools/qontract_cli.py

```diff
@@ -1083,41 +1085,6 @@ def ocm_aws_infrastructure_access_switch_role_links(ctx):
         print_output(ctx.obj["options"], by_user[user], columns)
 
 
-@get.command()
-@click.pass_context
-def clusters_egress_ips(ctx):
```

Comment:
@TGPSKI @bkez322 , can you confirm you do not use that qontract-cli command on your side?

The same info (and other) is available in `clusters-network`

Reply:
I'll go ahead and merge. We can always reinstate that if needed (adding the vpcid lookup that is now needed), but I believe  you'd get covered with `clusters-network`

Reply:
i don't think there is an app-interface-output is fedramp (yet?)